cmd/generate-project-list/main.go

```diff
@@ -44,6 +48,14 @@ func relevantProject(name string) bool {
 	return strings.HasPrefix(name, "jekyll") || additionalProjectNames[name]
 }
 
+func projectGemName(name string) string {
+	if gemName, exist := gemNames[name]; exist {
```

Comment:
Nit: if we're not using this map anywhere else, I'd rather see this method use a case statement. Easier to scan since it's all in one place.